Library/Homebrew/download_strategy.rb

```diff
@@ -312,6 +323,11 @@ def _fetch
     curl @url, "-C", downloaded_size, "-o", temporary_path
   end
 
+  def actual_url(url)
+    m = Utils.popen_read("curl", "-I", "-L", url).match /^Location: (.+)$/
```

Comment:
Yes that's what I thought to but I did test it, it worked. Have a reproducible example?

Sent from my iPhone

> On 16 May 2015, at 17:28, Xu Cheng <notifications@github.com> wrote:
> 
> In Library/Homebrew/download_strategy.rb:
> 
> > @@ -312,6 +323,11 @@ def _fetch
> >      curl @url, "-C", downloaded_size, "-o", temporary_path
> >    end
> >  
> > +  def actual_url(url)
> > +    m = Utils.popen_read("curl", "-I", "-L", url).match /^Location: (.+)$/
> This won't work. We cannot add -L. If you test it, you will see why.
> 
> —
> Reply to this email directly or view it on GitHub.
>